src/pytest/includes.py

```diff
@@ -0,0 +1,7 @@
+
+import sys
+import os
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "../../deps/readies"))
+import paella
```

Comment:
I'd rather have this optional as well..

Reply:
I admit that the motivation for this piece of code is the ability to use BB() in test code to trigger a breakpoint (in either pdb or pudb), when invoking RLTest like this: PYDEBUG=1 RLTest
(without PYDEBUG=1, BB() is ignored.)
So the purpose here is making test debugging ubiquitous. In light of this, making the user explicitly enable the import would somehow miss the target. 
I suggest we go with the original version, along with documentation which I'll add to this PR.

Reply:
Do a try for `import paella`